### ISSUE TITLE
Unity Lewis Number Transport Option

### ISSUE THREAD

**Issue by wandadars** [Cantera/cantera#507]
I noticed a dead pull request with an implementation for a unity lewis number transport option. I have also seen that this feature has been asked about multiple times on the Users' Group. I'm currently working to take the original pull-request work that was done and integrate the suggestions that @speth made. I may have some questions, so I made this issue. Let me know if there is a better way to do this.

**Comment by wandadars** [comment]
Some questions that I had while starting on this:

*I had the UnityLewisTransport class derive from the MixTransport class as @speth suggested. This seemed to really simplify the work required to implement the changes to get the updated mixture diffusion coefficient.

1.) Should the binary diffusion coefficients method also be changed to set them all to alpha/rho*cp? I ask because I'm not sure if it would make sense for a user to ask for unity Lewis number and then use the multicomponent diffusion fluxes option in the 1D solver. 

2.) When I create the UnityLewisTransport class, what do I edit in the TransportFactory to add it as an option? I didn't see a clear selection tree for picking the derived classes.

3.) There is still the issue of how to override the diffusion coefficient for the molar fluxes i.e. what should those be set to compared to setting the mass ones to alpha/rho*cp.

**Comment by awehrfritz** [comment]
@wandadars I had implemented the unity Lewis number transport model a few months back, but never found the time to prepare the pull request. I moved that code to an own branch now and submitted a pull request, see #510. The implementation is minimal and I have tested this in several flame calculations.

Regarding your questions on the implementation:
Ad. 1): No, I believe (and that's how I implemented it) that the diffusion coefficient returned by the transport model should correspond to that model.

Ad. 2): This can (and should in my opinion) all be implemented in the header file only. Thus the changes in the transport factory are minimal, see my implementation.

Ad. 3): I am unclear on what should be returned by `getMixDiffCoeffsMole` and  `getMixDiffCoeffsMass`. This would require more work to derive the respective expressions. I am also not sure on how much sense this would make anyway.
